PublicResolver.lll

```diff
@@ -0,0 +1,225 @@
+;;; ---------------------------------------------------------------------------
+;;; @title A public resolver; only allows the node's owner to set its address.
+;;; @author Daniel Ellison <daniel@syrinx.net>
+
+(seq
+
+  ;; --------------------------------------------------------------------------
+  ;; Constant definitions.
+
+  ;; Booleans.
+  (def 'true 1)
+  (def 'false 0)
+
+  ;; Memory layout.
+  (def 'scratch-one 0x00) ; Scratch space.
+  (def 'scratch-two 0x20) ; More scratch space.
+  (def 'return-code 0x40) ; Boolean returned to calling contract.
```

Comment:
`scratch-two` didn't end up being used either. I'll remove both `return-code`and `scratch-two` and then renumber everything.